src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const img = document.querySelector('img.spider');
+const div = document.querySelector('div, wall');
+
+img.style.top = div.clientHeight / 2 - img.clientHeight / 2 + 'px';
+img.style.left = div.clientWidth / 2 - img.clientWidth / 2 + 'px';
```

Comment:
The calculation for centering the image horizontally seems incorrect. It should subtract half of the image's width (img.clientWidth / 2) from half of the div's width (div.clientWidth / 2).